packages/google_maps_flutter/google_maps_flutter/example/ios/.gitignore

```diff
@@ -0,0 +1,32 @@
+*.mode1v3
```

Comment:
we probably don't need this as well

Reply:
I will revert the `example`. I think `flutter create` was run inside the `example` dir, and it added a lot of stuff that is not needed for this PR.